src/SFA.DAS.Admin.Aan.Web/Controllers/ManageMembers/RemoveMemberController.cs

```diff
@@ -59,6 +59,14 @@ public async Task<IActionResult> Index([FromRoute] Guid id, SubmitRemoveMemberMo
         PostMemberStatusModel postMemberStatusModel = new PostMemberStatusModel();
         postMemberStatusModel.Status = submitRemoveMemberModel.Status;
         await _outerApiClient.PostMemberLeaving(id, adminMemberId, postMemberStatusModel, cancellationToken);
-        return RedirectToRoute(SharedRouteNames.NetworkDirectory);
+
+        return RedirectToAction("RemoveMemberConfirmation");
+    }
+
+    [HttpGet]
+    public ActionResult RemoveMemberConfirmation()
+    {
+        ViewBag.NetworkDirectoryLink = Url.RouteUrl(SharedRouteNames.NetworkDirectory)!;
```

Comment:
You should have used a view model here to add the link to and bind the view with the view model. Also you should be able to test this then.